api/core/project/v1/project.proto

```diff
@@ -17,6 +17,7 @@ message Project {
   // Allows an organization to populate any information they see fit that does not fit our schema
   map<string, google.protobuf.Value> data = 5;
   ProjectDependencies dependencies = 6;
+  Pagerduty pagerduty = 7;
```

Comment:
should we nest this in an oncall block or something? just in case we expand for future non-pagerduty

Reply:
Or we could do an `Alerts` message with any type of information which could also include oncall? Or maybe the word `Alerts` is too specific.